spec/unit/mutant/ast/named_children_spec.rb

```diff
@@ -17,26 +25,42 @@
     let(:node) { s(:node, node_foo, node_bar, node_baz) }
 
     describe 'generated methods' do
+      specify 'are private by default' do
+        %i[
+          foo
+          bar
+          remaining_children
+          remaining_children_indices
+          remaining_children_with_index
+        ].each do |name|
+          expect(klass.private_instance_methods.include?(name)).to be(true)
```

Comment:
I do not like the truthyness check it internally delegates to. I do want a `#equal?`. On the return.